src/generated/resources/data/wotr/tags/item/socketable_main_hand_slot_weapon.json

```
@@ -0,0 +1,18 @@
+{
+  "values": [
+    "minecraft:wooden_sword",
+    "minecraft:stone_sword",
+    "minecraft:iron_sword",
+    "minecraft:golden_sword",
+    "minecraft:diamond_sword",
+    "minecraft:netherite_sword",
+    "minecraft:bow",
+    "minecraft:crossbow",
+    "minecraft:wooden_axe",
+    "minecraft:stone_axe",
+    "minecraft:iron_axe",
+    "minecraft:golden_axe",
+    "minecraft:diamond_axe",
+    "minecraft:netherite_axe"
```

We didn't ask you to completely remove the hybrid tag, only to not-give it any special modifiers but instead include it in the other tags (like how `#wotr:socketable` is made of other tags): 
```suggestion
    "#wotr:socketable_main_hand_slot_hybrid"
```
And where did the mace and trident go?

---

they weren't in there